.github/workflows/percy-prepare.yml

```diff
@@ -42,6 +55,8 @@ jobs:
         run: |
           echo ${{ inputs.pr_number }} > pr_num.txt
           echo ${{ inputs.commitsh }} > pr_head_sha.txt
+          echo "Contents of artifact"
+          tree .
```

Comment:
Is this needed, or was it just for debugging?

Reply:
It helped for debugging, but I think it will also help us identify any filestructure problems that are effecting Percy more quickly in the future. It took Josh and I a little while to figure out that we needed to add `_macros/` into the artifact